### DISCUSSION TITLE
Punkt vs Komma

### DISCUSSION THREAD

**Discussion by Federahannes** [TobiasPSP/OReilly#18] [Fragen]
Kapitel: 2 ; Thema: Operatoren

Ich bitte vorab um Entschuldigung für die Trivialität meiner Frage (bin halt absoluter PS-Beginner).

Im Beispiel auf der S. 123 (erste Auflage) wird der Wert '9.4' mit Punkt-Notation eingegeben.
Die Ergenisse '$ergebnis1' , .. ,'$ergebnis3' erfolgt allerdings mit Komma-Notation (vgl. Screenshot).

Ist das Normal? Warum ist das so? Und wie kann man als "Programmierer" wissen, ob man ggf. irgendwelche Ländereinstellungen von Benutzern apriori berücksichtigen muss? 

Gruß & Dank
![Screenshot Frage Punkt vs Komma](https://user-images.githubusercontent.com/110595225/211146870-d782b432-5b3e-4d96-9597-402cfa4ba030.PNG)

**Comment by Federahannes** [comment]
Ich bitte um Entschuldigung. Ich sehe gerade auf S. 127 die Antwort auf meine Frage. 
(wie kann ich meine Frage zurückziehen / löschen?)

**Comment by TobiasPSP** [comment]
:-) Alles gut! Es beruhigt mich, da ich leider erste soeben die Benachrichtigung für einen neuen Beitrag hier erhalten habe - zwei Jahre zu spät. Gut also, dass sich die Frage schnell beantworten ließ.
